src/Symfony/Component/Yaml/Escaper.php

```diff
@@ -48,7 +52,7 @@ class Escaper
      */
     public static function requiresDoubleQuoting($value)
     {
-        return preg_match('/'.self::REGEX_CHARACTER_TO_ESCAPE.'/u', $value);
+        return strlen($value) !== strcspn($value, implode('', self::$escapees));
```

Comment:
This would force escaping values containing `"`, `/` and `\` that weren't escaped before. Does it matter ?